src/test/java/io/confluent/kafkarest/unit/ConsumerManagerTest.java

```diff
@@ -216,10 +235,10 @@ public void testConsumerMaxBytesResponse() throws InterruptedException, Executio
     // response, not all of it is returned.
     final List<ConsumerRecord<byte[], byte[]>> referenceRecords
         = Arrays.<ConsumerRecord<byte[], byte[]>>asList(
-        new BinaryConsumerRecord(null, new byte[512], 0, 0),
-        new BinaryConsumerRecord(null, new byte[512], 1, 0),
-        new BinaryConsumerRecord(null, new byte[512], 2, 0),
-        new BinaryConsumerRecord(null, new byte[512], 3, 0)
+        new BinaryConsumerRecord(null, new byte[511], 0, 0), // Don't use 512 as this happens to fall on boundary
```

Comment:
This test passed when using 512 because the max bytes was 1024.  The code in ConsumerReadTask used a >= to check so 2 messages triggered the exit.  Dropping to 511, two messages should still trigger the exit (because the next message takes it over the max bytes) but the original code didn't exit (and in fact it locked up due to the mock time bug).